backend/subroker/migrations/0001_initial.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.7 on 2021-11-06 08:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Ott',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('ott', models.CharField(choices=[('Watcha', 'Watcha'), ('Netflix', 'Netflix'), ('Tving', 'Tving'), ('Youtube', 'Youtube'), ('Disney', 'Disney'), ('CoupangPlay', 'CoupangPlay'), ('Wavve', 'Wavve')], max_length=15)),
+                ('membership', models.CharField(choices=[('Basic', 'Basic'), ('Standard', 'Standard'), ('Premium', 'Premium')], max_length=10)),
```

Comment:
maybe memberships would vary ott by ott, is it ok to have just basic, standard, premium?